### PR TITLE
Expose Hive table properties via system table$properties table

### PR DESCRIPTION
**Pull request by zhenxiao**
Cherry-pick of https://github.com/prestosql/presto/commit/e7478a3200c26f9ca312f9b29c752de4e4843b7e

Co-authored-by: Zhenxiao Luo <zluo@twitter.com>

```
== RELEASE NOTES ==

Hive Changes
* Expose Hive table properties via system table$properties table
```

### CODE REVIEW

**Comment by wenleix** [comment]
@qqibrow : Can you sign the CLA? it should just be one click for the link in https://github.com/prestodb/presto/pull/14043#issuecomment-581083714 . Thank you ! 😃

**Comment by qqibrow** [comment]
@zhenxiao @wenleix that's done. Thank you!

File: presto-hive/src/main/java/com/facebook/presto/hive/security/SystemTableAwareAccessControl.java [hunk 1]
```diff
@@ -26,17 +26,16 @@
...
 import static java.util.Objects.requireNonNull;
 
-public class PartitionsAwareAccessControl
+public class SystemTableAwareAccessControl
         extends ForwardingConnectorAccessControl
 {
     private final ConnectorAccessControl delegate;
 
-    public PartitionsAwareAccessControl(ConnectorAccessControl delegate)
+    public SystemTableAwareAccessControl(ConnectorAccessControl delegate)
```
Comment (wenleix) [inline presto-hive/src/main/java/com/facebook/presto/hive/security/SystemTableAwareAccessControl.java]: Curious why this part of change is missing in this file: https://github.com/prestosql/presto/commit/e7478a3200c26f9ca312f9b29c752de4e4843b7e#diff-a8c5fae2119a67c33cf42d8306cd2f16R70-R72 -- just want to double check if it's not accidentally missing :)

Comment (zhenxiao) [reply presto-hive/src/main/java/com/facebook/presto/hive/security/SystemTableAwareAccessControl.java]: not missing. Some difference between the two files

**Pull request merged**